lib/rspec/mocks/minitest_integration.rb

```diff
@@ -0,0 +1,15 @@
+require 'rspec/mocks'
+
+Minitest::Expectation.class_eval do
```

Comment:
If we implement the suggestion I left below about putting the setup/verify/teardown integration here as well then I think we should make this file work on all Minitest 5 versions, not just on Minitest 5.6+.  I believe that `Minitest::Expectation` is only defined on 5.6+ so we're probably going to have to make this conditional on `Minitest::Expectation` existing as a constant.